data-transform/src/main/java/com/linkedin/data/transform/filter/request/MaskCreator.java

```diff
@@ -57,6 +57,23 @@ public static MaskTree createPositiveMask(Collection<PathSpec> paths)
     return createMaskTree(paths, MaskOperation.POSITIVE_MASK_OP);
   }
 
+  /**
+   * Create a positive mask for the given set.
+   *
+   * @param pathSpecSet the set that should be in the mask
+   * @return a {@link MaskTree}
+   */
+  public static MaskTree createPositiveMask(PathSpecSet pathSpecSet)
+  {
+    if (pathSpecSet.isFullProjection()) {
```

Comment:
braces on new line?

Reply:
Is there a code style I can refer to?

Reply:
Good q. I think it has been left braces on next line ever since restli first version.
Although I don't think there is strict rules regarding coding style, (for example, the line width)
I just don't want to see in the future there will be many places where mixed type is happening inside the repo

Reply:
Yeah, AFAIK the Rest.li code style isn't documented formally. It's just based on convention